Stratis.Bitcoin.Features.LightWallet/LightWalletFixedFeePolicy.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Wallet;
+using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.Features.LightWallet
+{
+    /// <summary>
+    /// Light wallet fixed fee policy used for stratis network.
+    /// </summary>
+    public class LightWalletFixedFeePolicy : IWalletFeePolicy
+    {
+        /// <summary>Logger instance for this class.</summary>
+        private readonly ILogger logger;
+
+        /// <summary>Fixed fee rate to use for this policy.</summary>
+        public FeeRate TxFeeRate { get; set; }
+
+        /// <summary>
+        /// Constructor for the light wallet fixed fee policy.
+        /// </summary>
+        /// <param name="loggerFactory">The factory for building logger instances.</param>
+        public LightWalletFixedFeePolicy(ILoggerFactory loggerFactory)
+        {
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+
+            // TODO: For now default is hardcoded, should probably be configurable 
+            this.TxFeeRate = new FeeRate(10000);
```

Comment:
FYI the TODO good point, maybe this should go in to the Network class and be set when building the network as it will apply to every network a min fee policy and always be mandatory.

Also one more idea, what do you think when we take code from X we can post the link to the place in X that we copied from?

Reply:
RE: minfee policy For fullnode wallet looks like min tx fee is 1000 for bitcoin and 10000 for strat.  What do you think about putting fallbackfee into network the same way?

Reply:
No NodeSettings is not distinguishable by network we would have to put an if condition (I see you already added the if condition)
I think its not a big deal to add it in NBitcoin.
So you propose to add MinFee and FallBackFee in to network in NBitcoin? sounds goo to me.
I would suggest we do that and then your PR will change a bit.
I am happy to commit this as it is now and then make the other changes:
- Commit as is
- Add MinFee  and FallBackFee  to Network in NBitcoin
- Bring changes to the fullnode and change in the correct places

Sounds good?

Reply:
Sounds good. Right now I created a network extension method for the default fees which has the nasty "if" in it.  I agree it would be cleaner if the defaults were stored directly in the NBitcoin networks.